Modix.Bot/Modules/EmojiStatsModule.cs

```diff
@@ -159,7 +159,7 @@ private async Task<EmbedBuilder> BuildEmojiStatEmbedAsync(SortDirection sortDire
                     .Append($" ({"use".ToQuantity(emojiStat.Uses)})")
                     .Append($" ({percentUsage.ToString("0.0")}%),")
                     .Append($" {perDay.ToString("0.0/day")}")
-                    .Append(canAccess ? string.Empty : $" (`:{emoji.Name}:`)")
+                    .Append(canAccess ? string.Empty : $" ({Format.Url($":{emoji.Name.Replace("_", "\\_")}:", emoji.Url)})")
```

Comment:
`$@`? that way you wouldn't have to deal with replacing

Reply:
Fixed